app/coffee/Features/Project/ProjectEditorHandler.coffee

```diff
@@ -18,40 +14,42 @@ module.exports = ProjectEditorHandler =
 			spellCheckLanguage: project.spellCheckLanguage
 			deletedByExternalDataSource : project.deletedByExternalDataSource || false
 			deletedDocs: project.deletedDocs
+			members:     []
 
-		if options.includeUsers
-			result.features =
-				collaborators: -1 # Infinite
-				versioning: false
-				dropbox:false
-				compileTimeout: 60
-				compileGroup:"standard"
-				templates: false
-				references: false
+		result.features = # defaults
+			collaborators: -1 # Infinite
+			versioning: false
+			dropbox:false
+			compileTimeout: 60
+			compileGroup:"standard"
+			templates: false
+			references: false
+		
+		owner = null
+		for member in members
+			if member.privilegeLevel == "owner"
+				owner = member.user
+			else
+				result.members.push @buildUserModelView member.user, member.privilegeLevel
+		if owner?
+			result.owner = @buildUserModelView owner, "owner"
 
-			if project.owner_ref.features?
-				if project.owner_ref.features.collaborators?
-					result.features.collaborators = project.owner_ref.features.collaborators
-				if project.owner_ref.features.versioning?
-					result.features.versioning = project.owner_ref.features.versioning
-				if project.owner_ref.features.dropbox?
-					result.features.dropbox = project.owner_ref.features.dropbox
-				if project.owner_ref.features.compileTimeout?
-					result.features.compileTimeout = project.owner_ref.features.compileTimeout
-				if project.owner_ref.features.compileGroup?
-					result.features.compileGroup = project.owner_ref.features.compileGroup
-				if project.owner_ref.features.templates?
-					result.features.templates = project.owner_ref.features.templates
-				if project.owner_ref.features.references?
-					result.features.references = project.owner_ref.features.references
+		if owner?.features?
+			if owner.features.collaborators?
+				result.features.collaborators = owner.features.collaborators
+			if owner.features.versioning?
+				result.features.versioning = owner.features.versioning
+			if owner.features.dropbox?
+				result.features.dropbox = owner.features.dropbox
+			if owner.features.compileTimeout?
+				result.features.compileTimeout = owner.features.compileTimeout
+			if owner.features.compileGroup?
+				result.features.compileGroup = owner.features.compileGroup
+			if owner.features.templates?
+				result.features.templates = owner.features.templates
+			if owner.features.references?
+				result.features.references = owner.features.references
```

Comment:
Might find this code a bit cleaner with _.defaults(settings.defaultFeatures ,owner.features) (notTested)